### ISSUE TITLE
Error while sending files scanned by email

### ISSUE THREAD

**Issue by locopine** [GM-Script-Writer-62850/PHP-Scanner-Server#45]
When I send the scanner by email, I receive the following message:

> `{"title":"404 Not Found","message":"No scan or domain specified"}`

Can someone tell me what it's about?

Thankful.

**Comment by GM-Script-Writer-62850** [comment]
I assume you got that via the your browsers debug info
Your from field when sending a email needs to be your email address including the `@emailSite.org` part
when that field it altered the page looks up the email servers config based on the emailSite.org part if there is no at website part in the field the server would return that data

The only other way to get that data from the server is to have null data file names for the data
if that is what is happening i need to know what page you are on eg just scanned/edited/all scans page and mult-file or not and if it is mult-file something is very screwy as there should be a empty array to get something else to happen
there is a hidden input field above the from field you can see with inspect element
you can view the content of this field with this line in a JS scratchpad 
```
(document.email.file||document.email.json).value // show field content
(document.email.file||document.email.json).value==null // Print true or false (true mean there is a problem)
```

**Comment by locopine** [comment]
Thank you very much for responding!

As I see it, even the name of the file being written in the hidden field that you commented on, the file is not actually sent to the "`tmp`".

I debug before the conditional arrival that checks `$ _POST` and `JSON` and both do not satisfy the condition (`true`).

In short, as soon as I get some time, I do a few more checks.

Finding the reason or any other question arises, I return the contact.

Again, very grateful.

**Comment by GM-Script-Writer-62850** [comment]
While if you read my coding it looks like it will put the file in /tmp/ it really does not
with systemd apache2 has a rather annoying feature called privateTmp that makes /tmp/ a fake one which prevents sharing data between programs via /tmp in the name of security
this should not be a issue for this project until you are trying to debug something and [this happens](https://i.imgur.com/2sQV6nx.gif)
I became aware of this `feature` when i was having issues passing data between python scripts and apache2 on my raspberry pi

That said i think i found the issue that is likely happening to you, on my local system i seem to be missing a comma on line 1521 of `res/main.js`

**Comment by GM-Script-Writer-62850** [comment]
I looks like i overlooked a comma on [2018/06/05](https://github.com/GM-Script-Writer-62850/PHP-Scanner-Server/commit/1e8d6cb7930367538832ea04be37ba9d03c04e4a#diff-b3a14d341f1147be50536fdf9df048bf)

**Comment by GM-Script-Writer-62850** [comment]
Please clear your browser's recent history your email login will be in your history as a result of that bug

**Comment by locopine** [comment]
It's cool that I participated, even indirectly, in resolving the **bug**!

I believe it to be my first participation in an application I tested.

If it were not for the current rush, which is why I was forced to go in search of something of the genre to be incorporated into a project, I would not have reached that application, I would not have come across the bug. This point was exactly what I needed !!!

Success.

If I can be useful in anything else, give me a ring ...

Thank you very much.
